src/BenchmarkDotNet/Detectors/Cpu/Windows/WmicCpuInfoParser.cs

```diff
@@ -52,6 +52,9 @@ internal static CpuInfo Parse(string? wmicOutput)
             ? Frequency.FromMHz(sumMaxFrequency / processorsCount)
             : null;
 
+        if (string.IsNullOrEmpty(processorName))
```

Comment:
Theoretically, a situation where we fail to determine the processorName but successfully identify other properties is valid. It is preferable to return null if `wmicOutput` is null or empty.